lib/bin/lwm2m_carrier/os/lwm2m_os.c

```diff
@@ -412,30 +412,32 @@ void lwm2m_os_sms_client_deregister(int handle)
 }
 
 /* Download client module abstractions. */
+static char dlc_buf[2048];
```

Comment:
Not everyone using the carrier library wants or needs this much, so it's a bit of a disadvantage to no longer have a configurable size for the buffer. We might need a new entry in lwm2m_carrier/Kconfig for that purpose.

Reply:
Added.